ElectricSaw.java

```diff
@@ -0,0 +1,36 @@
+package ua.lviv.iot.algo.part1.lab1;
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
+@Getter
+@ToString
+
+public class ElectricSaw extends Saw {
+    private int batteryCharge;
+    public double charge(int charge){
+        this.batteryCharge = batteryCharge + charge;
+        return this.batteryCharge;
+    }
+    public boolean start(){
+        setWorking(true);
+        return getWorking();
+    }
+    public boolean stop(){
+        setWorking(false);
+        return getWorking();
+    }
+    public String convertingToString(){
```

Comment:
name of the method should start with infinitive verb, better change it to "convert" without ing